core/lib/libtomcrypt/cbc.c

```diff
@@ -128,11 +128,12 @@ TEE_Result crypto_aes_cbc_alloc_ctx(struct crypto_cipher_ctx **ctx)
 #endif
 
 #if defined(_CFG_CORE_LTC_DES)
+#if !defined(CFG_CORE_SE05X)
```

Comment:
would be nice to find a smoother way. This seems a bit intrusive in LTC. Maybe playing with the configs (here `_CFG_CORE_LTC_DES`)?

Reply:
so somewhere in the build structure if CFG_CORE_SE05X try to undef _CFG_CORE_LTC_DES? I think that would make the code more obscure but I can try doing that

Reply:
see comment above about using a crypto driver.

Reply:
Yes better to avoid/minimize changes to LTC. `_CFG_CORE_LTC_XXX` are defined in `core/crypto.mk` from `CFG_CRYPTO_XXX` when the crypto library is LTC, I think it would be the right place to take `CFG_CORE_SE05X` into account.